src/docs/reference/modules/Media.AmazonS3/README.md

```diff
@@ -18,20 +18,22 @@ The following configuration values are used by default and can be customized:
 {
    "OrchardCore": {
        "OrchardCore_Media_AmazonS3": {
-           // Your AWS S3 Bucket name
-           "BucketName": "",
+           // If you have aws cli installed and configured you may just specify profile name
+           "Profile": "",
+           // In case your aws profiles are located not in default place
+           "ProfilesLocation": "",
+           "Region": "",
```

Comment:
```suggestion
           // If you have AWS CLI installed and configured you may just specify a profile name.
           "Profile": "",
           // In case your AWS profiles are located not in the default place.
           "ProfilesLocation": "",
           "Region": "",
```